tsconfig.types.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "./tsconfig.base.json",
   "compilerOptions": {
+    "incremental": false,
     "declaration": true,
```

Comment:
Just to understand, if it's only impacting types / doc generation, why isn't this single change enough?

Reply:
`incremental: false` here fixed the problem only for types inferred within the project. However, `logging.appenders` unfolds to `@kbn/config-schema` type https://github.com/elastic/kibana/pull/76890/files#diff-66a08ea686ee74b5fa19fbb89c8c7803L366 `@kbn/config-schema/tsconfig.json` has incremental builds enabled. I decided that I'd rather specify an explicit type for this use-case than disable incremental builds for the whole package.

Reply:
Ok, thanks for the explanation